tls/server-callback.c

```diff
@@ -200,21 +204,24 @@ int Server(word16 port)
 	            err_sys("Fatal error : respond: write error");
 	    }
 
-	    if (n < 0)
+	    if (n < 0) {
 	        err_sys("Fatal error :respond: read error");
+        }
```

Comment:
It looks like lines prior to this one have hard tabs. Please see pull request #47. This should resolve that.